quickstart/app/helper/function.php

```diff
@@ -0,0 +1,20 @@
+<?php
+function GetCategory($mang,$parent,$shift,$active)
+{
+    foreach($mang as $row)
```

Comment:
thêm cách giữa for/foreach/white/if/v.v... và đóng mở ngoặc. Mở ngoặc nhọn sẽ viết ngang dòng với các trường hợp này và chỉ xuống dòng mới đối với function hay class